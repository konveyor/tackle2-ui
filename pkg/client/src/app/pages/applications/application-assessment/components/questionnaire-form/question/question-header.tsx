import React from "react";
import { StackItem } from "@patternfly/react-core";

export interface QuestionHeaderProps {
  children?: React.ReactNode;
}

export const QuestionHeader: React.FC<QuestionHeaderProps> = ({
  children = null,
}) => {
  return <StackItem className="pf-u-pb-sm">{children}</StackItem>;
};
