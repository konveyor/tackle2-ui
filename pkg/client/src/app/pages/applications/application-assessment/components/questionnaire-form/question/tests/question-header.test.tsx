import React from "react";
import { shallow } from "enzyme";
import { QuestionHeader } from "../question-header";

describe("QuestionHeader", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<QuestionHeader>content</QuestionHeader>);
    expect(wrapper).toMatchSnapshot();
  });
});
