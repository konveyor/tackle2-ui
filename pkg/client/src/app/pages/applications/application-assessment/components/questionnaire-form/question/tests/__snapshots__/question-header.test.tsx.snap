// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionHeader Renders without crashing 1`] = `
<StackItem
  className="pf-u-pb-sm"
>
  content
</StackItem>
`;
